Relationsinstitutet/myko
# Make DIY activity part of the regular activity flow.

Refactored the code such that the DIY-form is shown in the same modal as regular activities.
Also, the form will be hidden after it has successfully been submitted.

## How to test it
1. Go to http://localhost:3000/activities/tillverka-aktivitet
2. Click "Gör nu"
3. Error case: add some whitespace in the fields and submit -> Error message should be shown
4. Success case: add some text in the fields and submit -> A success message should be shown
5. Go to http://localhost:3000/settings -> Entries for "Tillverka aktivitet" should be listed under "Genomförda aktiviteter"

@mipodi I don't think I have access to see the entries that are stored in Google forms, so could you please verify that it's working as expected while testing? (There are no changes regarding that code logic in this PR, but just to be on the safe side. 🙂 )
